lib/browser/api/app.ts

```diff
@@ -65,6 +65,8 @@ if (process.platform === 'darwin') {
     setDockMenu(menu)
   }
   app.dock!.getMenu = () => dockMenu
+  const dockBounce = app.dock!.bounce
+  app.dock!.bounce = (type = 'informational') => dockBounce(type)
```

Comment:
Can we move this to CPP instead, I want to move away from these JS side hacks as it makes things like context isolated electron harder in the future.  Using either a cpp default (not sure how that works with mate) or using `mate::Arguments` or using a `base::Optional<std::string>`